features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapProcessorImpl.java

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
You don't need to create a complex TrapProcessor class; you only need to create a mock object for testing. Remove this class and create an inner class inside TrapdHandlerDefaultIT.java instead.